core/src/scss/components/local-footer/_local-footer.scss

```diff
@@ -236,13 +236,29 @@
 }
 
 
-// List heading element styles.
+// List and content block heading element styles.
 // ------------------------------------------------
 .su-local-footer__list-heading {
   font-size: 18px;
   line-height: 140%;
 }
 
+.stanford-local-footer {
+  &.su-wysiwyg-text {
+    h2 {
+      font-size: 18px;
+      line-height: 140%;
+    }
+
+    h3,
+    h4,
+    h5 {
+      font-size: 16px;
+      line-height: 140%;
```

Comment:
@jdwjdwjdw If you could please just convert the font sizes from px to rem unit, eg, 18px = 1.8rem, this should be good to go. We're trying to avoid px units since those doesn't scale up if user changes their font size setting in the browsers.

Reply:
@yvonnetangsu sounds good, thanks - should be good to re-review now.